client/payment-details/transaction-breakdown/index.tsx

```diff
@@ -0,0 +1,319 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { filter, flatten, join } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useTimeline } from 'wcpay/data';
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	CardFooter,
+	Flex,
+	FlexItem,
+} from '@wordpress/components';
+import { TimelineItem, TimelineFeeRate } from 'wcpay/data/timeline/types';
+import Loadable, { LoadableBlock } from 'components/loadable';
+import { formatCurrency } from 'multi-currency/interface/functions';
+import './style.scss';
+
+interface PaymentTransactionBreakdownProps {
+	paymentIntentId: string;
+}
+
+const PaymentTransactionBreakdown: React.FC< PaymentTransactionBreakdownProps > = ( {
+	paymentIntentId,
+} ) => {
+	const { timeline, timelineError, isLoading } = useTimeline(
+		paymentIntentId
+	);
+
+	let captureEvents: TimelineItem[] = [];
+
+	if ( timeline ) {
+		captureEvents = filter( timeline, function ( item: TimelineItem ) {
+			return item.type === 'captured';
+		} );
+	}
+
+	let captureEvent: TimelineItem | undefined;
+	if ( captureEvents.length > 0 ) {
+		captureEvent = captureEvents[ 0 ];
+	}
+
+	if (
+		undefined === captureEvent ||
+		undefined === captureEvent.transaction_details ||
+		undefined === captureEvent.fee_rates
+	) {
+		return <div />;
+	}
+
+	const formattedStoreAmount =
+		formatCurrency(
+			captureEvent.transaction_details.store_amount,
+			captureEvent.transaction_details.store_currency
+		) +
+		' ' +
+		captureEvent.transaction_details.store_currency;
+
+	const formattedCustomerAmount =
```

Comment:
Minor: This can be simplified by using a template literal instead of string concatenation. This makes the code more readable and modern.

```suggestion
	const formattedStoreAmount = `${formatCurrency(
	captureEvent.transaction_details.store_amount,
	captureEvent.transaction_details.store_currency
)} ${captureEvent.transaction_details.store_currency}`;

```